kogitoq/gadaq/src/main/java/life/genny/gadaq/live/data/InternalConsumer.java

```diff
@@ -127,11 +131,54 @@ public void getEvent(String event) {
 
 		log.info("Received Event : " + SecurityUtils.obfuscate(event));
 
+		// Quick search
+		if(filter.isQuickSearchDropdown(msg)) {
+			filter.sendListQuickSearchDropdown(msg);
+			return;
+		}
+
 		// If the event is a Dropdown then leave it for DropKick
 		if ("DD".equals(msg.getEvent_type())) {
 			return;
 		}
 		events.route(msg);
+
+		// Saved search buttons
+		if(filter.isFilterBtn(msg)) {
```

Comment:
This should go in the `Events.java` Class

Reply:
Moved to events